char.zp

```diff
@@ -1,59 +1,179 @@
-(define (char:cmp? cmp a b) "compares two chars with a compare option cmp"
+(define (char:cmp? cmp a b)
+  "compares two chars with a compare option cmp.
+  params:
+   - compare option
```

Comment:
for exemple?
- compare_option: a compare function [string] (required)
What format?